CMakeLists.txt

```diff
@@ -27,6 +32,31 @@ set_target_properties(segyio-static PROPERTIES COMPILE_FLAGS "-fPIC")
 add_library(segyio-shared SHARED ${SOURCE_FILES})
 set_target_properties(segyio-shared PROPERTIES OUTPUT_NAME segyio CLEAN_DIRECT_OUTPUT 1)
 
+
+if(BUILD_PYTHON)
+    find_package(PythonInterp)
+    find_package(PythonLibs REQUIRED)
+
+    if (PYTHONLIBS_FOUND)
+        include_directories(${PYTHON_INCLUDE_DIRS})
+        add_library(_segyio SHARED python/segyio/_segyio.c ${SOURCE_FILES})
+
+        if(${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
+            set_target_properties(_segyio PROPERTIES LINK_FLAGS "-undefined dynamic_lookup")
+        else()
+            set_target_properties(_segyio PROPERTIES LINK_FLAGS "-Xlinker -export-dynamic")
+        endif()
+
+        set_target_properties(_segyio PROPERTIES PREFIX "" SUFFIX ".so")
```

Comment:
The `.so `suffix applies to Linux and MacOS. Windows may use `.pyd` files but we do not support Windows yet.